$OpenBSD$

Add stationToken to addFeedback request: fixes thumbs up/down and
move song
(upstream git commit c4330c3f65b2eacd3433ba19b1483f6704303205)

--- src/libpiano/request.c.orig	Mon May 14 09:58:35 2012
+++ src/libpiano/request.c	Mon May 14 10:00:03 2012
@@ -175,8 +175,11 @@ PianoReturn_t PianoRequest (PianoHandle_t *ph, PianoRe
 			
 			assert (reqData != NULL);
 			assert (reqData->trackToken != NULL);
+			assert (reqData->stationId != NULL);
 			assert (reqData->rating != PIANO_RATE_NONE);
 
+			json_object_object_add (j, "stationToken",
+					json_object_new_string (reqData->stationId));
 			json_object_object_add (j, "trackToken",
 					json_object_new_string (reqData->trackToken));
 			json_object_object_add (j, "isPositive",
